rte/main.go

```diff
@@ -83,6 +83,13 @@ func main() {
 		os.Exit(0)
 	}
 
+	if !parsedArgs.Resourcemonitor.RefreshNodeResources {
```

Comment:
this is an interesting idea and avoid us a RTE PR. But we still need a new option (can be specific of rte/main.go, we have already some custom options), because repurposing existing option is not fine.

Reply:
or just use `--no-publish`?